src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,65 @@
+import { Link, useParams } from 'react-router-dom';
+
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { path } = useParams();
+
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  const male = sex === 'm';
```

Comment:
1. let's create variable outside component for "m" because it's magic value
```jsx
const MALE = 'm';
```

2. Naming convention can be different from project to project but in general, we name boolean variables with prefix `is` or `has`

```jsx
const isMale = sex === MALE;
```